encord/objects/ontology_labels_impl.py

```diff
@@ -1158,12 +1159,17 @@ def _to_encord_data_unit(self, frame_level_data: FrameLevelImageGroupData) -> Di
         ret: Dict[str, Any] = {}
 
         data_type = self._label_row_read_only_data.data_type
-        if data_type == DataType.IMG_GROUP:
-            data_sequence: Union[str, int] = str(frame_level_data.frame_number)
-        elif data_type in (DataType.VIDEO, DataType.DICOM, DataType.IMAGE):
-            data_sequence = frame_level_data.frame_number
-        else:
-            raise NotImplementedError(f"The data type {data_type} is not implemented yet.")
+        match data_type:
```

Comment:
CI will tell you but match statements are Python 3.10 whereas SDK needs to support 3.8 :(
I think I made the exact same mistake at some point

Reply:
You can configure your IDE's syntax level to 3.8 to catch these!

Reply:
Ahh thanks for that! Have updated it to use if else's (which sadly don't look as nice T.T)